lib/tasks/hourglass.rake

```diff
@@ -44,15 +44,17 @@ namespace :hourglass do
   task :monthly_report, [:date] => :environment do |_, args|
     Rails.logger.silence do
       date = args[:date] ? Date.parse(args[:date]) : Date.current.last_month
-      puts GenerateReport.call!(range: date.all_month).output
+      tags = (ENV["TAGS"] || ENV["TAG"]).to_s.split(",")
```

Comment:
Sounds good. While I like the concept of rake variables, in practice the syntax and usage just feels awkward. ENV variables are fine with me.